OrbitGl/TimeGraph.h

```diff
@@ -141,6 +145,7 @@ class TimeGraph {
   const TextBox* FindDown(TextBox* from);
 
   Color GetThreadColor(ThreadID tid) const;
+  std::string GetManualInstrumentationString(uint64_t string_address) const;
```

Comment:
nodiscard

Reply:
Let's discuss the use of [[nodiscard]] with the team.

Reply:
Done.